openpype/tools/ayon_sceneinventory/models/site_sync.py

```diff
@@ -42,8 +42,16 @@ def get_site_provider_icons(self):
 
         if not self.is_sync_server_enabled():
             return {}
-        site_sync = self._get_sync_server_module()
-        return site_sync.get_site_icons()
+        site_sync_addon = self._get_sync_server_module()
+        icon_paths = {}
+        icon_defs = site_sync_addon.get_site_icons()
+        for provider, icon_def in icon_defs.items():
+            icon_def_type = icon_def.get("type")
+            if icon_def_type != "path":
+                continue
+            icon_paths[provider] = icon_def["path"]
+
+        return icon_paths
```

Comment:
If this is to fix QIcon converstion them please change it back. Fix is already in https://github.com/ynput/OpenPype/pull/6123
```suggestion
        site_sync_addon = self._get_sync_server_module()
        return site_sync_addon.get_site_icons()
```